android/app/src/googleplay/java/org/fossasia/openevent/utils/ImageUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.fossasia.openevent.utils;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.DrawableRes;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.content.res.ResourcesCompat;
+import android.support.v4.graphics.drawable.DrawableCompat;
+
+import com.google.android.gms.maps.model.BitmapDescriptor;
+import com.google.android.gms.maps.model.BitmapDescriptorFactory;
+
+/**
+ * Created by rohanagarwal94 on 6/7/17.
+ */
+public class ImageUtils {
+
+    public static BitmapDescriptor vectorToBitmap(Context context, @DrawableRes int id, int color) {
```

Comment:
Thing is markers cant directly use vector drawables which was the purpose of creating it in the first place.